esmvaltool/cmorizers/data/downloaders/datasets/esacci_sea_surface_salinity.py

```diff
@@ -0,0 +1,53 @@
+"""Script to download ESACCI-SOS."""
```

Comment:
Since this is still work in progress in #1832, I would not make the downloader and the formatter part of this PR. However, you may want to save the files (commits) for a future integration of this cmorizer.